src/components/command-bar/commands/search/helpers/generate-suggested-pages.tsx

```diff
@@ -87,7 +87,7 @@ const generateBasicSuggestedPages = (productSlug: ProductSlug) => {
  * link to the Tutorials Library.
  */
 const EXTRA_PAGES: Record<
-	Exclude<ProductSlug, 'sentinel'>,
+	Exclude<ProductSlug, 'sentinel' | 'well-architected-framework'>,
```

Comment:
What did this change fix?

Reply:
same comment as above, this was a type error